app/posts/manage-teacher-training-applications/2021-09-20-emailing-candidate-when-the-status-of-their-conditions-has-been-updated.md

```diff
@@ -34,15 +34,16 @@ Dear ((name))
 
 ((provider)) has updated the status of your conditions for ((course)).
 
-They’ve marked the following condition as met:
+They’ve marked the following conditions as met:
 
+- ((condition))
```

Comment:
Did you mean to change these, or was this a branching issue?

Reply:
Where has this come from? This wasn't in the post...